morphsnakes.py

```diff
@@ -202,6 +202,55 @@ def circle_level_set(image_shape, center=None, radius=None):
     return res
 
 
+def ellipsoid_level_set(image_shape, center=None, scale_factor=0.9):
+    """Create a ellipsoid level set with binary values.
+
+    Parameters
+    ----------
+    image_shape : tuple of positive integers
+        Shape of the image
+    center : tuple of positive integers, optional
+        Coordinates of the center of the ellipsoid given in (row, column).
+        If not given, it defaults to the center of the image.
+    scale_factor : float, optional
+        Scale factor for the ellipsoid, in [0, 1].
+        If set to 1, the ellipsoid is circumscribed in image of the given
+        `image_shape`. It defaults to 0.9.
+
+    Returns
+    -------
+    out : array with shape `image_shape`
+        Binary level set of the ellipsoid with the given `center`
+        and `scale_factor`.
+
+    See also
+    --------
+    circle_level_set
+    """
+
+    if center is None:
+        center = tuple(i // 2 for i in image_shape)
+
+    if len(image_shape) == 2:
+        rx, ry = center
+        phi = scale_factor - np.fromfunction(
```

Comment:
The result of the lambda function is the squared distance to the center. Shouldn't `scale_factor` be `scale_factor**2`?

Reply:
I think it should, yes. In fact, I think we can find a better name than `scale_factor` if we keep this implementation.